src/test/java/eu/europeana/metis/sandbox/test/utils/SolrContainerInitializerIT.java

```diff
@@ -0,0 +1,48 @@
+package eu.europeana.metis.sandbox.test.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.testcontainers.containers.SolrContainer;
+import org.testcontainers.utility.DockerImageName;
+
+public class SolrContainerInitializerIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SolrContainerInitializerIT.class);
+    static final SolrContainer solrContainer;
+    public static final String SOLR_VERSION = "solr:9.0.0";
```

Comment:
we checked the version we are using at the moment, please modify it to `7.7.3`

Reply:
Updated